extension/src/editors/process-editor/inscription-view/new-user-dialog.ts

```diff
@@ -2,15 +2,32 @@ import * as vscode from 'vscode';
 import { executeCommand } from '../../../base/commands';
 import { InscriptionActionHandler, SendInscriptionNotification } from './action-handlers';
 import { InscriptionActionArgs } from '@axonivy/inscription-protocol';
+import { DialogType, dialogTypes } from '../../../project-explorer/new-user-dialog';
 
 export class NewHtmlDialogActionHandler implements InscriptionActionHandler {
   actionId = 'newHtmlDialog' as const;
   async handle(actionArgs: InscriptionActionArgs, sendInscriptionNotification: SendInscriptionNotification) {
     const tabInput = vscode.window.tabGroups.activeTabGroup.activeTab?.input;
-    if (tabInput instanceof vscode.TabInputCustom) {
-      await executeCommand('ivyProjects.addNewHtmlDialog', tabInput, [], actionArgs.context.pid);
-      sendInscriptionNotification('dataChanged');
-      sendInscriptionNotification('validation');
+    if (!(tabInput instanceof vscode.TabInputCustom)) {
+      return;
     }
+    const dialogType = await this.collectDialogType();
+    if (!dialogType) {
+      return;
+    }
+    const command = dialogType === 'Form' ? 'ivyProjects.addNewFormDialog' : 'ivyProjects.addNewHtmlDialog';
+    await executeCommand(command, tabInput, [], actionArgs.context.pid);
+    sendInscriptionNotification('dataChanged');
+    sendInscriptionNotification('validation');
+  }
+
+  private collectDialogType(): Promise<DialogType | undefined> {
+    return vscode.window.showQuickPick(
+      dialogTypes.filter(t => t !== 'JSFOffline'),
```

Comment:
can we filter only if the caller is a UserTask?

Reply:
the current action payload does not contain this information.

Reply:
let's drop the filter than ... and just allow any view-technology to be used on dialog creation.